docs/features/tracing.rst

```diff
@@ -1,7 +1,7 @@
 Tracing
 =======
 
-Ocelot providers tracing functionality from the excellent `Butterfly <https://github.com/ButterflyAPM>`_ project. 
+Ocelot providers tracing functionality from the excellent `Butterfly <https://github.com/liuhaoyang/butterfly>`_ project. 
```

Comment:
So I updated the link to https://github.com/liuhaoyang/butterfly

Reply:
lol...too early in the morning for me :(